docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+* rewrites routes containing `redirectTo` for correct redirection.
+* creates a new route (used when routing away from the module):
+** matched when none of the defined routes were matched.
+** displays nothing (for a period of time when the user routes between Microfrontends).
+--
++
+// TODO: link to statsWith
+The `startsWith` method uses Microfrontend information, saved in the route's data, to remove already used parts from consideration when matching routes within the module.
++
+// TODO: link to lib
+To create your own matchers, please consider using `@onecx/angular-webcomponents` library.
++
+For technologies other than Angular, it is recommended to:
++
+--
+* Use Microfrontend information from CurrentMfeTopic to only use relevant parts of the URL for routing and redirect correctly.
+* Ensure routing away from the module is not causing side-effects.
+--
+
+Configuration::
+All services utilizing HttpClient used within the Microfrontend's module need to know how to make requests to external resources. Depending on the configuration of the Workspace, they need to take that context into consideration when creating a URL for those resources.
++
+A service might want to call `deployment_url/bff/search` by default. With this call being made, MFE App will need to access the BFF. When the Application's path within the Workspace is `mfe/example` the call has to be made to `deployment_url/mfe/example/bff/search`.
++
+// TODO: Link to Topic
+The `apiConfigProvider` presented in the example is utilizing `PortalApiConfiguration` class as `Configuration` for the services. It is listening to the CurrentMfeTopic changes and overwriting the basePath accordingly to the received message and all services use that information to construct a valid URL.
++
+For technologies other than Angular, it is recommended to:
++
+--
+* Listen to CurrentMfeTopic changes and overwrite services configuration to ensure the correct resource is accessed.
+--
+
+==== Remote Component bootstrapping and configuration
+The following requirements need to be fulfilled when exposing Remote Components using the Webcomponent method:
+
+. Webpack configuration needs to expose the *main.ts* file of the Remote Component.
+. The **main.ts** file of the Remote Component needs to bootstrap the component asynchronously.
+// TODO: Correct link to exact resource for Angular issues
+. Component bootstrap
+.. has to cover Angular issues (for Angular based Remote Components only)
+.. needs to register the component as a Custom Element in the Custom Elements Registry.
+.. needs to adapt routing so Shell registers routing happening within the Remote Component.
+. Component class needs to use the Remote Component initialization mechanism.
+
+Below is a Remote Component setup example (Angular based) for using the Webcomponent expose method is presented. Each file's content is going to be explained in detail with important context information for technologies other than presented.
```

Comment:
Below is a Remote Component setup example (Angular based) for using the Webcomponent expose method ~~is presented~~.